client/hack/update-crd.sh

```diff
@@ -39,7 +39,7 @@ then
   exit 1;
 fi
 
-$CONTROLLER_GEN crd:trivialVersions=true,preserveUnknownFields=false paths=${SCRIPT_ROOT}/apis/volumesnapshot/v1beta1
+$CONTROLLER_GEN crd:crdVersions=v1,trivialVersions=true,preserveUnknownFields=false paths=${SCRIPT_ROOT}/apis/volumesnapshot/v1beta1
```

Comment:
preserveUnknownFields=false is now redundant?

Reply:
@yuxiangqian can you explain why it is redundant now?  What does preserveUnknownFields=false mean?

Reply:
preserveUnknownFields=false is there in v1beta1 apiextensions to prevent adding a CR with unknown/undefined fields persisted. However, that's no longer needed in v1 apiextension according to [this](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#field-pruning).

Reply:
Sure.

Reply:
removed the `preserveUnknownFields=false` flag